presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -128,6 +137,49 @@ public Password()
         }
     }
 
+    private static class Roles
```

Comment:
same for CLI?

Reply:
Yes, but I would prefer not to do it with this PR.